src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackProcessor.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest
+
+private val logger = KotlinLogging.logger {}
+
+@Service
+class CallbackProcessor {
+    /**
+     * returns
+     *  - a list of errors
+     *  - *or* an empty list  if it successfully attaches the supplementary evidence.
+     */
+    fun processEventType(type: String, callback: CallbackRequest): List<String> {
```

Comment:
probably over kill when we will only ever have 1 event type in orchestrator

Reply:
hmmm ... sure.
Are you suggesting to do event/type validation in the controller and specialise the "service" to a attachEventProcessor ?

Reply:
specialise the service to do what you need today, and not something too generic, as we aren't expecting to add more callbacks.

not blocking this PR though, but imo its too generic and does nothing atm